<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>105</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Chrome</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>85</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>WI-FI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_99">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>182</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Get passwords</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>441</width>
      <height>351</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>source</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>location</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>username</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>password</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>390</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>490</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clean</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>153</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Find password</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>441</width>
      <height>201</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>source</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>location</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>username</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>password</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>441</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Find</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>309</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Write location here</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>440</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>540</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>540</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Write location here</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>949</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
